android/app/src/main/AndroidManifest.xml

```diff
@@ -29,5 +29,16 @@
                 <category android:name="android.intent.category.LAUNCHER"/>
             </intent-filter>
         </activity>
+        <provider
+    android:name="vn.hunghd.flutterdownloader.DownloadedFileProvider"
```

Comment:
what about a linter config?

Reply:
Good idea! Android XML files don't make up a large portion of the codebase, but at least for Dart files, this would certainly be a very useful feature. Opened #46